tests/conftest.py

```diff
@@ -1,8 +1,8 @@
 import pytest
-from responses import RequestsMock
 
 from app.factory import create_app
-
+from flask import url_for
+from responses import RequestsMock
```

Comment:
3rd party modules should be imported before local modules. The order of imports should be:
* core modules (eg: `os`, `urllib`)
* 3rd party modules (eg: `flask`, `responses`)
* local modules (eg: `app.main`, `tests.oidc_testbed`)

with a single blank line between each set and two blank lines after the last import